public/assets/icons/burger.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const BurgerIcon = () => (
+  <>
+    <svg
```

Comment:
It's nice you've created created components for the SVGs. I'm curious why you took this approach instead of treating them like `jpgs` or `pngs` though?

I was saying the same thing to Susanna yesterday, and that was that I think we can just have `.svg` files and reference them in the Next image component, with the source path being e.g.  `/public/icon/burger.svg`.

Let me know what you think. If you go this route you'll have to modify the styles a little bit, but it shouldn't be much.

Reply:
I did it because the tests were failing. Do we need to mock image set-up? and i saw Susanna did it, so yeah followed.

Reply:
Do you remember what the errors were saying?